### PR TITLE
chore(policies): moves affected DPPs into overview

### PR DESCRIPTION
**Pull request by kleinfreund**
Moves the affected DPPs tab content into the overview sections and removes the tabs.

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

File: src/app/policies/components/PolicyConnections.vue [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
       :error="error"
     />
 
-    <EmptyBlock v-else-if="data === undefined || data.items.length === 0" />
+    <LoadingBlock v-else-if="data === undefined" />
+
+    <EmptyBlock v-else-if="data.items.length === 0" />
 
     <template v-else>
       <p
```
Comment (lahabana) [inline src/app/policies/components/PolicyConnections.vue:28]: Can we change this to at least an `<ul>` or something like this while we're touching this?

Comment (kleinfreund) [reply src/app/policies/components/PolicyConnections.vue]: Done.

File: src/app/policies/components/PolicyConnections.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <h2>Dataplanes</h2>
+  <h3>{{ t('policies.detail.affected_dpps') }}</h3>
```
Comment (johncowen) [inline src/app/policies/components/PolicyConnections.vue:2]: I think these things should be move into the `*View.vue`. We should try to never have `h*`s in components as it makes them not reusable. I'm pretty sure we could get rid of the `PolicyDetails` component entirely?

Comment (kleinfreund) [reply src/app/policies/components/PolicyConnections.vue]: I moved the heading into the PolicyDetails component, but I want to keep this split of *View and (mostly) presentational components.

**Review by lahabana — APPROVED** [review]
Feels like the PolicyConnections component could be inlined.

**Pull request merged**